### PR TITLE
AZP: fix command find grep for large count of files

### PR DESCRIPTION
**Pull request by avildema**
## What
CI: fix command find grep for large count of files

## Why ?
CI on Azure pipeline fail with 123 code on find grep

### CODE REVIEW

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -701,7 +701,7 @@ check_config_h() {
 	# Check if all .c files include config.h
 	echo "==== Checking for config.h files in directory $srcdir ===="
 
-	missing=`find $srcdir -name \*.c -o -name \*.cc | xargs grep -LP '\#\s*include\s+"config.h"'`
+	missing=`find $srcdir -name \*.c -o -name \*.cc -exec grep -LP '\#\s*include\s+"config.h"' {} \;`
```
Comment (hoopoepg) [inline contrib/test_jenkins.sh:704]: s/-LP/-LPH
to print file name

**Review by hoopoepg — APPROVED** [review]

**Comment by yosefe** [comment]
bot:pipe:retest

**Review by yosefe — APPROVED** [review]

**Review by yosefe — CHANGES_REQUESTED** [review]
I tested this (added C file without include config.h) and it doesn't work

**Review by hoopoepg — APPROVED** [review]

**Review by yosefe — APPROVED** [review]
thanks, now it works
can you pls squash?

**Pull request merged**